pkg/gardenlet/controller/shoot/shoot_control.go

```diff
@@ -484,8 +482,7 @@ func (c *Controller) reconcileShoot(logger *logrus.Entry, shoot *gardencorev1bet
 	}
 
 	durationUntilNextSync := c.durationUntilNextShootSync(o.Shoot.Info)
-	message := fmt.Sprintf("Scheduled next queuing time for Shoot in %s (%s)", durationUntilNextSync, time.Now().UTC().Add(durationUntilNextSync))
-	c.recorder.Event(o.Shoot.Info, corev1.EventTypeNormal, "ScheduledNextSync", message)
+	logger.Infof("Scheduled next queuing time in %s (%s) for Shoot %+v", durationUntilNextSync, time.Now().UTC().Add(durationUntilNextSync), o.Shoot.Info)
```

Comment:
The logger already has fields identifying the Shoot :)
```suggestion
	logger.Infof("Scheduled next queuing time in %s (%s)", durationUntilNextSync, time.Now().UTC().Add(durationUntilNextSync))
```